### DISCUSSION TITLE
Github student account -> teacher account

### DISCUSSION THREAD

**Discussion by raulviana** [community/community#57454] [GitHub Education]
### Hi everyone,

Hi. I had a student account, but I have already finished the course. Now I meet the conditions to claim a teacher account. How can I change the account from student to teacher?

**Comment by samus-aran** [comment]
Hi @raulviana 👋 

We can revoke your current student coupon if you would like. Let us know and we'll do so 👍

**Reply by raulviana** [reply]
Hi @samus-aran. 
Yes I would like to revoke my student coupon in order to apply to a teacher account afterwards. 
Many thanks.
